# Part 1 

### 1. Line 9 prints out 20.
### 2. Line 13 prints out 20 as well, because var could be accessed anywhere inside sumValues. 
### 3. Line 9 prints out 20.
### 4. Line 13 returns an error, "Uncaught ReferenceError: result is not defined", this is becuase let is declared inside the if block, and it's only accessable inside that block.
### 5. Line 9 returns an error because const could not be reassigned after it has been assigned. 
### 6. The code terminated before it reached line 13.

